descriptors/sqlserver/debezium-sqlserver-1.9.0.Alpha1.json

```diff
@@ -71,35 +71,6 @@
           "x-name" : "database.dbname",
           "x-category" : "CONNECTION"
         },
-        "database.names" : {
-          "format" : "list,regex",
-          "title" : "Databases",
-          "description" : "The names of the databases from which the connector should capture changes",
-          "type" : "string",
-          "x-name" : "database.names",
-          "x-category" : "CONNECTION"
-        },
-        "database.instance" : {
```

Comment:
same here?

Reply:
Gunnar suggested not including it.

Reply:
Yeah, let's skip it for now. We can add it later on if needed.